packages/amplify-graphql-auth-transformer/package.json

```diff
@@ -32,6 +32,7 @@
     "@aws-amplify/graphql-relational-transformer": "0.12.4",
     "@aws-amplify/graphql-transformer-core": "0.17.15",
     "@aws-amplify/graphql-transformer-interfaces": "1.14.9",
+    "@aws-cdk/aws-iam": "^1.181.0",
```

Comment:
Why the new dependency?

Reply:
It wouldn't let me push without the dependency addition

Reply:
What's the error that you're seeing and when does it occur?

Reply:
@ncarvajalc could you flip this to `~1.172.0`.